client/styles/components/sidebar/index.scss

```diff
@@ -308,6 +309,31 @@
     }
 }
 
+.PoweredBy{
+    height: 60.99px;
+    width: 171.43px;
+    position: absolute;
+    border-radius: 8px;
+    border-color: #F9F9F9;
+    font-size: 14px;
+    background-image: url("/static/img/powered-by.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-top: 10%;
+}
+.PoweredBy:hover{
```

Comment:
Why not combine this with the above class since the content is exactly the same?